superset-frontend/src/components/Form/LabeledErrorBoundInput.stories.tsx

```diff
@@ -20,29 +20,26 @@ import React, { useState } from 'react';
 import LabeledErrorBoundInput, {
   LabeledErrorBoundInputProps,
 } from './LabeledErrorBoundInput';
-
```

Comment:
It looks like some spacing got removed here on line 23 and also on lines 28, 37, 45, 64, and 70, can you put them back for readability?